luminoso_api/client.py

```diff
@@ -85,7 +90,7 @@ def connect(cls, url=None, token_file=None, token=None):
             with open(token_file) as tf:
                 token_dict = json.load(tf)
             try:
-                token = token_dict.get(urlparse(root_url).netloc)
+                token = token_dict[root_url]
```

Comment:
While the change from `get` here, so that it's possible to trigger a KeyError, makes sense, losing the `urlparse(...).netloc` does not.  The stored keys are things like `analytics.luminoso.com`, and not `https://analytics.luminoso.com/api/v5/`.